### DISCUSSION TITLE
I made some sliding block puzzle games for the picocomputer

### DISCUSSION THREAD

**Discussion by ceptimus** [picocomputer/community#100] [Q&A]
Actually, I ported most of the puzzles, and the game engine, from a Commodore Amiga version.

It's my first program for the picocomputer.  I'm still working on extra new puzzles, the automatic puzzle creator (a Python program), and the documentation.  I'll upload the source code (all in C) once I've tidied it up a little.

Bug reports, suggestions for improvements, criticism, and other feedback most welcome.

https://youtu.be/CTzmy6But1E?si=7svU3-_JABf_lvTE

https://github.com/ceptimus/RP6502PUZZ

**Comment by vruumllc** [comment]
Fantastic!  
Thanks so much for creating and posting. I liked your video too.

**Reply by JosVermoesen** [reply]
Thank you for sharing! You both do good work for the community of Rumbledethumps his open source project. I learn a lot of you guys

**Comment by WojciechGw** [comment]
Superb!
Thank You very much :)

**Comment by netzerohero** [comment]
Very nice!  Thank you for sharing your efforts!

**Comment by James-HoneyBadger** [comment]
nice one! pretty clever coding
